lblod/frontend-embeddable-notule-editor
# Update editor packages to latest versions

## Overview
This PR updates the editor packages to their latest stable versions:
- `@lblod/ember-rdfa-editor` 9.5.1
- `@lblod/ember-rdfa-editor-lblod-plugins` 16.2.0

Additionally, this PR also enables support for customizing row background colors.

### How to test/reproduce
- Start-up the test-app
- Visit the index route and insert a table
- Notice that the table has alternating row colors, you can customize these in the table options
- Ensure that you can customize the cell colors



### Checks PR readiness
